Marlin/ultralcd.cpp

```diff
@@ -1110,13 +1110,24 @@ static void lcd_control_volumetric_menu() {
 #ifdef HAS_LCD_CONTRAST
   static void lcd_set_contrast() {
     if (encoderPosition != 0) {
-      lcd_contrast -= encoderPosition;
-      lcd_contrast &= 0x3F;
+      #ifdef U8GLIB_LM6059_AF
+        lcd_contrast += encoderPosition;
```

Comment:
> patch to fix the contrast issue I submitted ever get merged?

No, I see that your patch (MarlinFirmware/MarlinDev#200) was never taken up again after you closed it. But no worries. Your fix makes sense to address recent questions about lcd contrast, which I couldn't answer (#3912). So I can go ahead re-make it for `RCBugFix` and make sure it gets into 1.1.0. The `MarlinDev` repository is going to be retired anyway, with development moving back to the main repo.